aiohttp/abc.py

```diff
@@ -146,3 +146,13 @@ def write_eof(self, chunk=b''):
     @abstractmethod
     def drain(self):
         """Flush the write buffer."""
+
+
+class AbstractAccessLogger(ABC):
+
+    def __init__(self, logger, log_format):
+        self.logger = logger
```

Comment:
`log_format` seems to be lost.

Reply:
Yes, but it is exactly how current `AccessLogger` works, as well default `log_format` is `AccessLogger. LOG_FORMAT` not sure is it needs to be stored as `self.log_format`. My idea was that, `logger` itself is only one required attribute, `log_format` is actually optional

Reply:
So, if I'm making own `AccessLogger` I should keep in mind, that super requires two arguments, one of which it will set to attributes and one that it will ignore? A bit not friendly behavior. The `AccessLogger` may have overloaded `__init__` to accept `log_format`, but if the `AbstractAccessLogger` doesn't uses it in anyway there is no point to keep it there.